internal/tui/model_list.go

```diff
@@ -56,6 +89,17 @@ func NewListModel(ctx context.Context, description string, repoUser string, clie
 	l.SetFilteringEnabled(false)
 	l.SetShowStatusBar(false)
 	l.SetShowPagination(false)
+	l.AdditionalShortHelpKeys = func() []key.Binding {
+		return listKeys
+	}
+	l.AdditionalFullHelpKeys = func() []key.Binding {
+		return listKeys
+	}
```

Comment:
```suggestion
	l.AdditionalShortHelpKeys = func() []key.Binding {
		return m.KeyMap().ShortHelp()
	}
	l.AdditionalFullHelpKeys = func() []key.Binding {
		return m.KeyMap().ShortHelp()
	}

	m.list = l
```

This removes the need for `listKeys`

Note that you'll need to bring this logic to after the model is initialized.